2fa-api/handlers/create.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../db");
+
+const create = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+    const { phone } = req.body;
```

Comment:
what if one or more of these don't exist?

what about validation? do they need to follow any sort of pattern? how would you verify that?

Reply:
It will hit the catch block and return an error. The error message can be more specific and indicate that a field is missing. However this should probably be validated on the frontend beforehand anyways.

Reply:
it should be validated in both.

you should have a regex parser to determine if its a good email, or something else.